src/core/connectors.js

```diff
@@ -1245,6 +1245,11 @@ const connectors = [{
 	matches: ['*://hqradio.ru/*'],
 	js: 'connectors/hqradio.js',
 	id: 'hqradio',
+}, {
+	label: 'Smooth FM',
+	matches: ['*://smoothfm.iol.pt/player/*'],
```

Comment:
Could you update the URL pattern to `*://smoothfm.iol.pt/player/*` to cover the main page too?

Reply:
Done.